projects/ngneat/cashew/src/lib/httpCacheLocalStorage.ts

```diff
@@ -0,0 +1,53 @@
+import { HttpResponse } from '@angular/common/http';
+import { Inject, Injectable } from '@angular/core';
+import { deleteByRegex } from './deleteByRegex';
+import { HttpCacheStorage } from './httpCacheStorage';
+import { HTTP_CACHE_CONFIG, HttpCacheConfig } from './httpCacheConfig';
+
+@Injectable()
+export class HttpCacheLocalStorage implements HttpCacheStorage {
+  private readonly storageKey: string;
+
+  constructor(@Inject(HTTP_CACHE_CONFIG) private config: HttpCacheConfig) {
+    this.storageKey = config.localStorageKey || 'httpCache';
+  }
+
+  has(key: string): boolean {
+    const storage = this.getLocalStorage();
+    return !!storage[key];
+  }
+
+  get(key: string) {
```

Comment:
When you fetch it the first time, you'll need to convert it into `new HttpResponse(value)`.